changelog.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 5
 
+### Version 5.0.14
+
+* `dockerform`: Parsing for adding external service to docker-compose
```

Comment:
The current release is only 5.0.12, and so this change will become part of 5.0.13.